java/adapter/jdbc/src/main/java/org/apache/arrow/adapter/jdbc/JdbcToArrow.java

```diff
@@ -63,6 +63,90 @@
  */
 public class JdbcToArrow {
 
+  /**
```

Comment:
these should be deprecated as well.  The user should always use the iterator approach.